doc/source/dev/contributor/quickstart_mac.rst

```diff
@@ -2,189 +2,101 @@
 
 .. _quickstart-mac:
 
-================================================
-Development environment quickstart guide (macOS)
-================================================
+=======================================================
+Development environment quickstart guide (MacOS)
+=======================================================
 
 This quickstart guide will cover:
 
-* setting up and maintaining a development environment, including installing
-  compilers and SciPy build dependencies
-* creating a personal fork of the SciPy repository on GitHub
-* using git to manage a local repository with development branches
-* performing an in-place build of SciPy
-* creating a virtual environment that adds this development version of SciPy to
-  the Python path
+* setting up and maintaining a development environment, including installing compilers and SciPy build dependencies;
+* creating a personal fork of the SciPy repository on GitHub;
+* using git to manage a local repository with development branches;
+* performing an in-place build of SciPy; and
+* creating a virtual environment that adds this development version of SciPy to the Python path
 
-in macOS.
-
-Its companion videos `Anaconda SciPy Dev: Part I (macOS)`_ and
-`Anaconda SciPy Dev: Part II (macOS)`_ show many of the steps being performed.
-This guide may diverge slightly from the videos over time, with the goal of keeping
-this guide the simplest, up-to-date procedure.
+in MacOS (Tested on 11.1).
 
 .. note::
 
-	This guide does not present the only way to set up a development environment;
-	there are many valid choices of Python distribution, C/Fortran compiler, and
-	installation options. The steps here can often be adapted for other choices,
-	but we cannot provide documentation tailored for them.
+	This guide does not present the only way to set up a development environment; there are many valid choices of Python distribution, C/Fortran compiler, and installation options. The steps here can often be adapted for other choices, but we cannot provide documentation tailored for them.
 
-	This guide assumes that you are starting without an existing Python 3 installation.
-	If you already have Python 3, you might want to uninstall it first to avoid
-	ambiguity over which Python version is being used at the command line.
+	This guide assumes that you are starting without an existing Python 3 installation. If you already have Python 3, you might want to uninstall it first to avoid ambiguity over which Python version is being used at the command line.
 
-.. _quickstart-mac-build:
+.. _quickstart-macos-build:
```

Comment:
This change is causing the doc build failure. Elsewhere it's looking for the original name, but can no longer find it.